sparse_ho/models/svm.py

```diff
@@ -23,49 +23,57 @@ class SVM(BaseModel):
     max_iter : int
         maximum number of epochs for the coordinate descent
         algorithm
-    tol : float
-        tolerance for the stopping criterion
+    estimator: instance of ``sklearn.base.BaseEstimator``
+        An estimator that follows the scikit-learn API.
     """
 
     def __init__(self, estimator=None, max_iter=100):
         self.estimator = estimator
         self.max_iter = max_iter
         self.dual = True
-        self.dr = None
+        self.dresiduals = None
```

Comment:
this attributes do not seem documented in the docstring.

also is residual y - X @ beta even in classification?

is SVM just for regression? could we call it SVC et SVR?

Reply:
it's not the quantity y - X @ beta for SVC / SVR it corresponds to the dual variables. 
Yes I can change the name of the model for SVC for classification
I will add the documentation

Reply:
is it too misleading to call it residuals for these models ? Up to a scaling factor the dual variables of the Lasso are also the residuals right ?

Reply:
Well we could name it dual for the SVM type models but the problem is that it appears in forward/implicit forward (algorithms) code which has to be generic for all model types. Do you have suggestions?

Reply:
dual_var sounds good to me if it's not too misleading for l1 type models.